docs/pages/install.md

```diff
@@ -0,0 +1,110 @@
+| `pelican_amd64.apk`            | APK package for AMD64 (x86_64) architecture. |
+| `pelican_arm64.apk`            | APK package for ARM64 architecture. |
+| `pelican_Darwin_arm64.tar.gz`  | Tarball for macOS systems running on ARM64 architecture. |
+| `pelican_Darwin_x86_64.tar.gz` | Tarball for macOS systems with x86_64 architecture. |
+| `pelican_Linux_arm64.tar.gz`   | Tarball for Linux systems with ARM64 architecture. |
+| `pelican_Linux_ppc64le.tar.gz` | Tarball for Linux systems with little-endian PowerPC architecture. |
+| `pelican_Linux_x86_64.tar.gz`  | Tarball for Linux systems with x86_64 (64-bit) architecture. |
+| `pelican_ppc64le.apk`          | APK package for little-endian PowerPC architecture, suitable for some PowerPC-based systems or specific Linux distributions. |
+| `pelican_Windows_x86_64.zip`   | ZIP archive for Windows systems with x86_64 (64-bit) architecture. **Note:** Pelican on Windows is still experimental and you may run into issues. |
+
+##### RPM, APK, or DEB?
+###### RPM (Red Hat Package Manager):
+You want to install a `.rpm` package if you are using a Red Hat-based Linux distribution system such as: Red Hat Enterprise Linux, CentOS, Fedora, or openSUSE.
+
+###### APK (Alpine Package Keeper):
+You want to install a `.apk` package if you are using an Alpine Linux system.
+
+###### DEB (Debian Package):
+You want to install a `.deb` package if you are using a Linux distribution system such as: Debian, Ubuntu, or something similar.
+
+##### What about tar and zip?
+If you want a more manual setup, you can download the `.tar.gz` or `.zip` files and extract the binary where you need it. However, the above packages are recommended for more inexperienced users.
+
+##### What version should I download?
+Our versions are built like so:
+For example: 7.1.2
+- 7 represents the latest major version release
+- 1 represents feature releases
+- 2 represents a bug fix/patch release
+
+We recommend you to use the latest feature or major release version so it includes the latest bug fixes and features.
+
+### 2. Extract the Binary
+Once the package has finished downloading, place it in your workspace
+
+##### RPM:
+To install RPM packages, you can use the dnf or yum package manager. Just make sure you have sudo access if you are not root:
+```console
+sudo yum install pelican-7.1.2-1.x86_64.rpm
+```
+Replace 'yum' with 'dnf' for dnf install
+
+##### DEB:
+To install Debian packages, you can use apt or dpkg package manager:
+```console
+sudo dpkg -i pelican-7.1.2-1_amd64.deb
+sudo apt-get install -f
+```
+Or
+```console
+sudo apt install ./pelican-7.1.2-1_amd64.deb
+```
+
+##### APK:
+To install APK packages on Alpine Linux, you can use the apk package manager:
+```console
+apk add pelican_amd64.apk
+```
+
+##### Tarballs:
+To install `.tar.gz` packages, you can extract with tar:
+```console
+tar -xzf pelican_Darwin_x86_64.tar.gz
```

Comment:
We should probably make a note here that extracting the tarballs and zip files doesn't actually _install_ pelican -- it just extracts the binary. I know you cover it later, but I'm personally a fan of being explicit about things like this.